src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -143,7 +144,7 @@ public void CopyPackageSignatures(string outputPath)
 
                 File.Copy(
                     Path.Combine(entry.Library.Path, hash),
-                    Path.Combine(outputPath, entry.Library.Identity.Name, entry.Library.Identity.Version.ToNormalizedString(), hash),
+                    Path.Combine(outputPath, entry.Library.Identity.Name, entry.Library.Identity.Version.ToNormalizedString(), hash).ToLower(),
```

Comment:
It looks like `outputPath` part will also be lowercased, which is not intended, right?